### DISCUSSION TITLE
How to prevent re-downloading of minecraft jar without having to select Offline mode?

### DISCUSSION THREAD

**Discussion by TheInsane101** [PrismLauncher/PrismLauncher#2971] [Q&A]
I'm playing with a small redstone mod that was made by Myren Eario in 1.9 and uses the old-fashioned way of installing mods by copying the classes into the minecraft.jar file, which gets erased when the Launch button redownloads the vanilla jar. 
Launching in Offline mode prevents this, but I shouldn't have to do it everytime I want to play instances that have mods installed the old fashioned way.

https://github.com/user-attachments/assets/0f9f335d-dfed-48ab-9212-261dac71350c

I'm sorry if this is a stupid or obvious question; I'm still new to the different ways in which Java mods are installed. Is there a way to prevent the erasing of the mods without having to click the drop-down on the launch button? 

Also, does the launcher have a GUI for installing old mods like this, or do I just do the regular process of editing the .jars and copying the mod code into them like it was done back in the day?

**Comment by TayouVR** [comment]
I am not entirely sure, but I believe those older mods should be installed via the "Add to Minecraft.jar" button, and that should keep it around. I don't have any mods that work that way here right now, so I can't exactly test it, but maybe give that a try.
